app/main.py

```diff
@@ -1,8 +1,25 @@
 class Person:
-    # write your code here
-    pass
+    people = {}  # Хранилище всех созданных людей
 
+    def __init__(self, name, age):
+        self.name = name
+        self.age = age
+        Person.people[name] = self  # Сохраняем объект для ссылок на других людей
 
-def create_person_list(people: list) -> list:
-    # write your code here
-    pass
+
+def create_person_list(people_data: list) -> list:
+    # Создаем объекты Person
+    for data in people_data:
+        Person(data["name"], data["age"])
+
+    # Устанавливаем связи husband/wife
+    for data in people_data:
+        person = Person.people[data["name"]]
+        if "wife" in data and data["wife"]:
+            person.wife = Person.people[data["wife"]]
+            Person.people[data["wife"]].husband = person
+        if "husband" in data and data["husband"]:
+            person.husband = Person.people[data["husband"]]
+            Person.people[data["husband"]].wife = person
```

Comment:
The code for setting relationships (husband and wife) assumes that each person can have both a husband and a wife. This might not be the intended behavior if the data is expected to have only one type of relationship per person. Ensure that the data structure and logic align with the intended use case. If a person can only have one spouse, consider adding checks to prevent setting both `husband` and `wife` attributes for the same person.